iml-manager-cli/src/server.rs

```diff
@@ -218,250 +218,396 @@ async fn wait_till_agent_starts(
     Ok(())
 }
 
-#[allow(clippy::cognitive_complexity)]
-pub async fn server_cli(command: ServerCommand) -> Result<(), ImlManagerCliError> {
-    match command {
-        ServerCommand::List => {
-            let hosts: ApiList<Host> = wrap_fut("Fetching hosts...", get_hosts()).await?;
+fn list_server(hosts: ApiList<Host>) {
+    tracing::debug!("Hosts: {:?}", hosts);
+
+    let table = generate_table(
+        &["Id", "FQDN", "State", "Nids"],
+        hosts.objects.into_iter().map(|h| {
+            vec![
+                h.id.to_string(),
+                h.fqdn,
+                h.state,
+                h.nids.unwrap_or_else(|| vec![]).join(" "),
```

Comment:
Have you tried:

```rust
h.nids.unwrap_or_default().join(" "),
```

Reply:
I didn't try that. I'll try changing that now.

Reply:
Looks like it works fine.